### DISCUSSION TITLE
Bilmece bilmece sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#222] [Zeka Sorusu] [answered]
Eğer bu bilmeceyi çözmeden önce çözdüğünüz bilmece, bu bilmeceden daha zorsa, bu bilmeceyi çözmeden önce çözdüğünüz bilmeceden sonra çözdüğünüz bilmece, bu bilmeceyi çözmeden önce çözdüğünüz bilmeceden daha zor mudur?

**Comment by Denizmly** [comment, marked as answer]
![20230621_111620.jpg](https://github.com/sonsuzus/sonsuzus/assets/114510784/b110c36f-435b-4003-81fe-b96e9204aa1d)



Cevap hayır 
Bu bilmece = 1 olsun
Bu bilmeceden önce çözdüğümüz bilmece = 2 olsun
2 ,1'den  daha zor
Bu bilmeceden önce çözdüğümüz bilmeceden sonra çözdüğümüz bilmece= 3

 Bilmece 1 ve 3 aynı bilmeceler

Yani 

Bilmece 3 bize bilmece 2'den sonra çözdüğümüz bilmeceyi söylemek istemiş.

O bilmece de bilmece 1'dir 

Bize sorduğu soru
Bu bilmece (1), bu bilmeceyi çözmeden önce çözdüğümüz bilmeceden(2) daha zor mudur? 

*Cevap: Hayır *

Bilmece 2 bilmece 1'den daha zordur.
